PowerEditor/installer/nativeLang/Malayalam.xml

```diff
@@ -0,0 +1,954 @@
+                    <Item id="6226" name="ഒന്നുമില്ല"/>
+
+                    <Item id="6227" name="വരി റാപ്"/>
+                    <Item id="6228" name="സ്വതേ"/>
+                    <Item id="6229" name="ചേരിചേരാ"/>
+                    <Item id="6230" name="ഇൻഡന്റ്"/>
+
+                    <Item id="6206" name="പ്രദർശിപ്പിക്കുക ലൈൻ നമ്പറുകൾ"/>
+                    <Item id="6207" name="ബുക്ക്മാർക്കുകൾ പ്രദർശിപ്പിക്കുക"/>
+                    <Item id="6208" name="ലംബമായ എഡ്ജ് കാണിക്കുക"/>
+                    <Item id="6209" name="നിരകൾ എണ്ണം: "/>
+                    <Item id="6234" name="വിപുലമായ സ്ക്രോളിംഗ് സവിശേഷത അപ്രാപ്തമാക്കുക
+(നിങ്ങൾ ടച്ച്പാഡ് പ്രശ്നം ഉണ്ടെങ്കിൽ)"/>
+
+                    <Item id="6211" name="ലംബമായ എഡ്ജ് ക്രമീകരണങ്ങൾ"/>
+                    <Item id="6212" name="ലൈൻ മോഡിൽ"/>
+                    <Item id="6213" name="പശ്ചാത്തല മോഡ്"/>
+                    <Item id="6214" name="നിലവിലെ വരി ഹൈലൈറ്റിംഗ് പ്രാപ്തമാക്കുക"/>
+					<Item id="6215" name="മിനുസമുള്ള ഫോണ്ട് പ്രാപ്തമാക്കുക"/>
+                    <Item id="6231" name="ബോർഡർ വീതി"/>
+                </Scintillas>
+
+                <NewDoc title="പുതിയ പ്രമാണം">
+                    <Item id="6401" name="ഫോർമാറ്റ്"/>
+                    <Item id="6402" name="വിൻഡോസ്"/>
+                    <Item id="6403" name="യുണിക്സ്"/>
+                    <Item id="6404" name="മാക്"/>
+                    <Item id="6405" name="എൻകോഡിംഗ്"/>
+                    <Item id="6406" name="ആൻസി"/>
+                    <Item id="6407" name="UTF-8"/>
+                    <Item id="6408" name="യുട്ടിഎഫ് 8 BOM ൽ ഉപയോഗിച്ച്"/>
+                    <Item id="6409" name="നന്മ UCS2 ബിഗ് എൻഡ്യൻ"/>
+                    <Item id="6410" name="നന്മ UCS2 ലിറ്റിൽ എൻഡ്യൻ"/>
+                    <Item id="6411" name="സ്ഥിര ഭാഷ:"/>
+                    <Item id="6419" name="പുതിയ പ്രമാണം"/>
+                    <Item id="6420" name="ആൻസി ഫയലുകൾ തുറക്കാൻ പ്രയോഗിക്കുക"/>
+                </NewDoc>
+
+                <DefaultDir title="സ്ഥിരസ്ഥിതി ഡയറക്ടറി">
+                    <Item id="6413" name="സ്വതേ ഡയറക്ടറി (തുറക്കുക / സൂക്ഷിക്കുക))"/>
+                    <Item id="6414" name="നിലവിലുള്ള രേഖ പിന്തുടരുക"/>
+                    <Item id="6415" name="അവസാനം ഉപയോഗിച്ചത് ഡയറക്ടറി ഓർക്കുക"/>
+                </DefaultDir>
+
+                <FileAssoc title="ഫയൽ അസോസിയേഷൻ">
+                    <Item id="4009" name="പിന്തുണയുള്ള വിപുലീകരണങ്ങൾ:"/>
+                    <Item id="4010" name="രജിസ്റ്റർ വിപുലീകരണങ്ങൾ:"/>
+                </FileAssoc>
+                <LangMenu title="ഭാഷാ മെനു">
+                    <Item id="6505" name="ലഭ്യമായ ഇനങ്ങൾ"/>
+                    <Item id="6506" name="അപ്രാപ്തമാക്കി ഇനങ്ങൾ"/>
+                    <Item id="6507" name="ഭാഷ മെനുവിൽ ഒതുക്കമുള്ള വരുത്തുക"/>
+                    <Item id="6508" name="ഭാഷ മെനു"/>
+                </LangMenu>
+
+                <TabSettings title="ടാബ് ക്രമീകരണങ്ങൾ">

```

Comment:
`TabSettings` is no longer separate. It is merged to `Language` . Move all the four entries to `Language` tag.

Reply:
@SinghRajenM  Done